src/components/todo-input/TodoInput.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <form class="input-model">
+    <v-text-field
+      class="text-placeholder"
```

Comment:
Please change the name of the `text-placeholder` class to something more descriptive. This is the actual todo input so you can name the class to `todo-input`